docs/instruments/instrumentserver.md

```diff
@@ -141,3 +146,82 @@ Changing things in the GUI will also be reflected in the code.
     Changing something from the GUI only changes the code if we are calling the parameter manager directly.
     If we store a value in a separate variable and then change the GUI, the value in the variable might not get update.
     Because of this, we always recommend to call the Parameter Manager directly instead of saving the values in variables.
+
+## Config Files
+
+The instrumentserver also contains functionality for config files. These config files contain set-up for QCoDeS instruments. These config files can then be used by the instrumentserver 
+on start-up to have the desired instruments already created without having to manually add them after starting the server.
+
+To open the instrument server with a config file, we can use the '-c' flag to specify a config file, followed by the path to the config file:
+
+```bash
+$ instrumentserver -c serverConfig.yml
+```
+
+
+Below is a sample config file to initialize a DummyInstrument and an Oxford Triton dilution refrigerator
+
+`instruments`: this field will contain each instrument you wish to create. Under this field should be the names of each instrument you are creating
+
+`type`: This specifies which QCoDeS driver the instrumentserver should use to create the instrument. Can either be a local file or a community-created driver
+
+`address`: the tcp/ip address of the instrument on the same network as the instrumentserver
+
+`port`: which port (of the instrument) to communicate with (can generally be found in the manual of the instrument)
+
+```yaml
+instruments:
+  dummy_instrument_random_number:
+    type: instrumentserver.testing.dummy_instruments.generic.DummyInstrumentRandomNumber
+    initialize: True
+  fridge_test:
+    type: labcore.instruments.qcodes_drivers.Oxford.triton.OxfordTriton
+    address: 192.168.999.999
+    init: 
+      port: 33576
+```
+
+This is the result of running the instrument server with this config file:
+
+![two_instruments](img/two_instruments.png)
+
+### Parameter Manager with Config Files
+
+The instrumentserver also comes with the virtual instrument Parameter Manager.
+The Parameter Manager allows us to store values in an instrument inside of the Instrumentserver, allowing access from any process or device in the same network.
+The idea of it is to have a single source of truth for parameters whose values change frequently, and it provides a GUI from which you can change the values and easily see what they are.
+
+The following is an example config file to add the parameter manager as a virtual instrument
+
+```yaml
+instruments:
+  parameter_manager:
+    type: instrumentserver.params.ParameterManager
+    initialize: True
+    gui:
```

Comment:
Maybe add the parameter manager with the custom GUI in the example before so you can mention how the instrumentserver supports custom gui's for instruments that need to be specified like this.